WordPress/src/main/java/org/wordpress/android/ui/reader/adapters/ReaderMenuAdapter.java

```diff
@@ -72,7 +114,6 @@ public View getView(int position, View convertView, ViewGroup parent) {
         holder.mText.setText(textRes);
         holder.mText.setTextColor(AppCompatResources.getColorStateList(convertView.getContext(), textColorRes));
         ColorUtils.INSTANCE.setImageResourceWithTint(holder.mIcon, iconRes, iconColorRes);
```

Comment:
Minor (🔍): Consider accessing this static function via `ColorUtils` directly. See IDE warning and apply quick fix.